src/RFC-0303_DanOverview.md

```diff
@@ -0,0 +1,189 @@
+registration deposit also serves as a Sybil prevention mechanism.
+
+Validator nodes are the bridge between the consensus layer and the Tari Virtual Machine (TVM).
+
+VNs are required to re-register periodically as a proof-of-liveness mechanism.
+
+Validator nodes must be able to
+1. interpret the instructions they receive from clients, identifying the contract code that the instruction refers,
+2. retrieve and deserialize the relevant input state,
+3. compute the output state that result from applying the contract logic to the input state, and
+4. reach consensus with its peers.
+
+Steps 1 - 3 are carried out in the Tari Virtual Machine (TVM).
+Step 4 is achieved by communicating with peers via the DAN consensus layer.
+
+[Thaums](#thaums-and-the-turbine-model) exist at the Validator node level, and VNs earn fees, in Thaums, for each 
+instruction that it aids in getting finalised.
+
+### DAN consensus layer
+
+The Cerberus BFT consensus algorithm runs on the consensus layer. This layer is completely ignorant of the 
+semantics of DAN smart contracts.
+
+This layer only cares that:
+* _only_ VNs that have registered on the base layer are participating in consensus.
+* VNs are self-organising into VN committees and are carrying out the rules of Cerberus and emergent HotStuff 
+  correctly.
+
+In particular, the consensus layer has _no idea_ whether an instruction's output is correct. If two-thirds (plus one)
+of the committee say that the results are correct, then consensus has been reached and the consensus layer is happy.
+
+### The Tari Virtual Machine
+
+The TVM is a WASM-based virtual machine designed to run Tari contracts.
+Contracts are composed of one or more Tari templates, glue code and a state schema. Tari Labs provides a Rust 
+implementation for TVM contracts, but in principle, other languages could implement the specification as well.
+
+The TVM is able to
+* load a contract.
+* provide a list of methods that the contract exposes.
+* Execute calls on the contract.
+* Persist and restore the state of the contract.
+
+### Thaums and the turbine model
+
+Thaums is a [unit of magic](https://discworld.fandom.com/wiki/Thaum). Thaums are used to power the DAN's economic 
+engine. Thaums are created with a one-way perpetual peg as described in [RFC-0320]. Briefly, Tari is burnt to create 
+Thaums, which are used to pay for the execution of instructions on the DAN. A portion of instruction fees are burnt 
+with every instruction to provide a constant source of demand for Thaums in the DAN.
+
+There is no peg out back to the base layer for Thaums. Thaum holders wishing to convert back to Tari will be able to 
+perform a submarine swap with a Tari seller. We also anticipate that exchanges will list the Thaum-Tari pair to 
+enable easy conversion of Thaums back to Tari.
+
+# Change Log
+
+| Date        | Change              | Author     |
+|:------------|:--------------------|:-----------|
+| 1 Nov 2022  | High-level overview | CjS77      |
```

Comment:
Some RFCs have this in ascending order, some in descending. Not sure what the standard should be

Reply:
I also noticed the lack of consistency and decided to go with it for now :)